tests/stress_tests/memcheck_tests/tests.cpp

```diff
@@ -106,3 +106,51 @@ INSTANTIATE_TEST_CASE_P(MemCheckTests, MemCheckTestSuite,
                         ::testing::ValuesIn(
                                 generateTestsParams({"devices", "models"})),
                         getTestCaseName);
+
+TEST_P(MemCheckTestSuite, inference_with_streams) {
+    const auto nstreams = 2;
+    log_info("Inference of InferRequest from network: \"" << model
+                                                          << "\" with precision: \"" << precision
+                                                          << "\" for device: \"" << device << "\""
+                                                          << "\" with streams: \"" << nstreams);
+    if ((device != "CPU") && (device != "GPU"))
+        throw std::invalid_argument("This device is not supported");
+
+    auto test_pipeline = [&] {
+        MemCheckPipeline memCheckPipeline;
+        Core ie;
```

Comment:
please, move IE objects after`config` and `key` creation

i.e. it should be like:
```
std::map<std::string, std::string> config;
config[device + "_THROUGHPUT_STREAMS"] = std::to_string(nstreams);

Core ie;
ie.GetVersions(device);
ie.SetConfig(config, device);
...
InferRequest inferRequest;
```